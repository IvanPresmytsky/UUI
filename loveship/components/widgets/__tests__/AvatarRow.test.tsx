import React from 'react';
import { renderer } from '@epam/uui-test-utils';
import { AvatarRow } from '../AvatarRow';

describe('AvatarRow', () => {
    it('should be rendered correctly', () => {
        const tree = renderer.create(<AvatarRow img="https://api.dicebear.com/7.x/pixel-art/svg?seed=Coco&radius=50&backgroundColor=b6e3f4" size="42" />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
