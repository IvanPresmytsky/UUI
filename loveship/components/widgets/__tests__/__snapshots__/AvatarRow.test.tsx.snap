// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AvatarRow should be rendered correctly 1`] = `
<div
  className="avatarRow"
>
  <img
    className="avatar avatarContent size-42"
    height="42"
    onError={[Function]}
    src="https://api.dicebear.com/7.x/pixel-art/svg?seed=Coco&radius=50&backgroundColor=b6e3f4"
    width="42"
  />
</div>
`;
